### PR TITLE
[CST-2165] Simplify admin view

### PR DESCRIPTION
**Pull request by ghost**
### Context
Simplify admin view. Reduce the amount of tabs for ECTs. 

Prototype: https://manage-training-for-ects-5c9ed99fc0a8.herokuapp.com/admin/ects/show

- Ticket: https://dfedigital.atlassian.net.mcas.ms/browse/CST-2165

### Changes proposed in this pull request

- simplify ECT participant admin page by reducing the amount of tabs and removing duplication
### Guidance to review
- Navigate through a EPC participants admin tabs, check everything works as expected and whether some important functionality is missing

### CODE REVIEW

File: app/views/admin/participants/_nav.html.erb [hunk 1]
```diff
@@ -7,27 +7,16 @@
...
-    Induction records
-  <% end %>
-
   <%= component.with_nav_item(path: admin_participant_declaration_history_path(@participant_profile)) do %>
-    Declaration history
+    Declarations
   <% end %>
 
   <%= component.with_nav_item(path: admin_participant_validation_data_path(@participant_profile)) do %>
     Validation data
```
Comment (TomBr0wn) [inline app/views/admin/participants/_nav.html.erb:15]: The nav  'Validation data' item is not on the prototype. Does it stay, or is it removed?

**Review by TomBr0wn — COMMENTED** [review]
This looks good to go. Nav items match the prototype and image on ticket 2165.

**Review by frankieroberto — APPROVED** [review]
Looks good to me! A big improvement - and can be further iterated on in future. 🙌 

Not a show-stopper, but could you wrap the summary lists in this HTML, to make them not quite full-width? I think that'd make them easier to read.

```html
<div class="govuk-grid-row">
  <div class="govuk-grid-column-three-quarters">
     [...]
  </div>
</div>

**Review by TomBr0wn — APPROVED** [review]
Changes to tabs matching prototype (Keeping DQT Matching in) 
Looking good.

**Review by tonyheadford — APPROVED** [review]
LGTM

**Pull request merged**